service/controller/v22/resource/s3bucket/create.go

```diff
@@ -29,27 +29,29 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		return microerror.Mask(err)
 	}
 
+	g := &errgroup.Group{}
+
 	for _, bucketInput := range createBucketsState {
-		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating S3 bucket %#q", bucketInput.Name))
+		bucketInput := bucketInput
 
-		{
-			i := &s3.CreateBucketInput{
-				Bucket: aws.String(bucketInput.Name),
-			}
+		g.Go(func() error {
```

Comment:
Rewrote the parallel execution. Now the whole bucket setup is executed in series again but multiple buckets are executed in parallel. That was necessary due to creation issues.